src/main/resources/sparql/select-agent-summaries.rq

```diff
@@ -0,0 +1,52 @@
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX premis: <http://www.loc.gov/premis/rdf/v3/>
+PREFIX rdac: <http://rdaregistry.info/Elements/c/>
+PREFIX dct: <http://purl.org/dc/terms/>
+PREFIX cat: <http://cat.nationalarchives.gov.uk/>
+PREFIX foaf: <http://xmlns.com/foaf/0.1/>
+PREFIX ver: <http://purl.org/linked-data/version#>
+PREFIX rdaa: <http://rdaregistry.info/Elements/a/>
+PREFIX time: <http://www.w3.org/2006/time#>
+PREFIX todo: <http://TODO/>
+PREFIX prov: <http://www.w3.org/ns/prov#>
+
+SELECT DISTINCT ?identifier ?type ?currentDescription ?label ?dateFrom ?dateTo ?depository ?versionTimestamp ?previousDescription
+WHERE {
+    ?identifier dct:type ?agentType .
+    
+    VALUES (?agentType ?type ?dateFromProperty ?dateToProperty) {
+    	(cat:corporate-body-concept "Corporate Body" rdaa:P50037 rdaa:P50038)
+    	(cat:person-concept "Person" rdaa:P50120 rdaa:P50121)
+  	}
+
+    ?identifier ver:currentVersion ?currentDescription .
+    OPTIONAL {
+	    {
+	    	?currentDescription rdaa:P50032 ?label .
+	    }
+	    UNION {
+	        ?currentDescription rdaa:P50292 ?label1  .
+	        ?currentDescription rdaa:P50291 ?label2  .
```

Comment:
Instead of `?label1` and `?label2` we should call these what they are, i.e. `?givenName` and `?surname`